arches_her/pkg/graphs/branches/Record Type.json

```diff
@@ -0,0 +1,146 @@
+{
+    "graph": [
+        {
+            "author": "",
```

Comment:
You need to add an author for this branch - preferably the author's name and organisation
(e.g. we have Philip Carlisle - Historic England for the ones Phil has created)

Reply:
Added in latest commit https://github.com/archesproject/arches-her/pull/1228/commits/3bcf0379505261a4a8fceaa2405365032f48a845